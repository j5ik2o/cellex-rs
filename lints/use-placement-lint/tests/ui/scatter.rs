use std::fmt;

fn foo() {}

#[path = "auxiliary/helpers_inline.rs"]
mod helpers;

use std::str::FromStr;

#[cfg(test)]
mod tests;

fn main() {}
