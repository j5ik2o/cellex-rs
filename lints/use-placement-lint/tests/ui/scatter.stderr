warning: `use` と `mod` 宣言はファイル冒頭にまとめて配置してください
  --> $DIR/scatter.rs:5:1
   |
LL | / #[path = "auxiliary/helpers_inline.rs"]
LL | | mod helpers;
   | |____________^
   |
   = help: この宣言を最初の `use` / `mod` ブロックより前に移動しましょう
   = note: AI向けアドバイス: コメントや属性を含めたまま、該当の `use` / `mod` ブロックをファイル上部へ移動して一箇所に集約してください。
   = note: `#[warn(use_mod_placement)]` on by default

warning: `use` と `mod` 宣言はファイル冒頭にまとめて配置してください
  --> $DIR/scatter.rs:8:1
   |
LL | use std::str::FromStr;
   | ^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: この宣言を最初の `use` / `mod` ブロックより前に移動しましょう
   = note: AI向けアドバイス: コメントや属性を含めたまま、該当の `use` / `mod` ブロックをファイル上部へ移動して一箇所に集約してください。

warning: `use` と `mod` 宣言はファイル冒頭にまとめて配置してください
  --> $DIR/scatter.rs:10:1
   |
LL | / #[cfg(test)]
LL | | mod tests;
   | |__________^
   |
   = help: この宣言を最初の `use` / `mod` ブロックより前に移動しましょう
   = note: AI向けアドバイス: コメントや属性を含めたまま、該当の `use` / `mod` ブロックをファイル上部へ移動して一箇所に集約してください。

warning: 3 warnings emitted

